website/common/script/ops/stats/allocate.js

```diff
@@ -24,9 +24,6 @@ module.exports = function allocate (user, req = {}) {
   if (user.stats.points > 0) {
     user.stats[stat]++;
     user.stats.points--;
-    if (stat === 'int') {
```

Comment:
Actually we've been discussing this internally and it's possible that this feature (dealloation) isn't meant to exist unless in some specific cases so I'd wait a bit to see what @Tressley and @SabreCat think

Reply:
This isn't deallocation, it is allocation?

Reply:
You're right, so the problem now is that the server is unchanged but anyone can pass the `ignoreManaPoints` parameter to the API and have this change affect the server side as well which we don't want to allow

Reply:
Would giving it as a third parameter `options` to allocate which is always set by the server and the client be better? I considered this first, but no other functions had options, so I didn't think it was necessary. Alternatively we can validate the request before passing it to allocate.

I don't feel like it is a huge issue, since I can't see any reason why someone would want to avoid adding 1 mana point to themselves by manually sending a request, but I see it is possibly not ideal.